data_acquisition/src/calc_center.R

```diff
@@ -0,0 +1,99 @@
+#' @title Calculate POI for polygons
+#' 
+#' @description
+#' Use polygon and 'point of inaccessibility' function (polylabelr::poi()) to 
+#' determine the equivalent of Chebyshev center, furthest point from every edge 
+#' of a polygon. POI function here will calculate distance in meters using the 
+#' UTM coordinate system and the POI location as Latitude/Longitude in WGS84 
+#' decimal degrees.
+#' 
+#' @param yaml contents of the yaml .csv file
+#' @param poly sfc object of polygon areas for acquisition
+#' @returns filepath for the .shp of the polygon centers or the message
+#' 'Not configured to use polygon centers'. Silently saves 
+#' the polygon centers shapefile in the `data_acquisition/in` directory path 
+#' if configured for polygon centers acquisition.
+#' 
+#' 
+calc_center <- function(poly, yaml) {
+  if (grepl("center", yaml$extent[1])) {
+    # create an empty tibble
+    poi_df <- tibble(
+      r_id = integer(),
+      poi_longitude = numeric(),
+      poi_latitude = numeric(),
+      poi_dist_m = numeric()
+    )
+    # create rowids for proper indexing
+    poly <- poly %>% 
+      rowid_to_column("r_id")
+    for (i in 1:length(poly[[1]])) {
+      poi_df  <- poi_df %>% add_row()
+      # grab one polygon
+      one_wbd <- poly[i,]
```

Comment:
nitpick: There's a few brackets that technically could use a space for formatting, not a big deal though

```suggestion
      one_wbd <- poly[i, ]
```

Reply:
Gotta catch 'em all! I think I corrected all of them. There are still a few formatting things that allude me, I appreciate your attention to detail!